src/driver/voodoo.ml

```diff
@@ -222,6 +179,8 @@ let process_package pkg =
     |> List.flatten
   in
   let libraries = meta_libraries @ non_meta_libraries in
+  let pkg_dir = top_dir pkg in
+  let doc_dir = Fpath.(pkg_dir / "doc") in
```

Comment:
The doc directory is back ? Is this to avoid breaking urls ?

Reply:
I chose not to change the layout on ocaml.org - mostly because it'd require a bunch of redesign work, and I don't have the capacity to do that.